packages/analytics-js-integrations/src/integrations/Gainsight_PX/nativeSdkLoader.js

```
@@ -0,0 +1,29 @@
+      hostName = 'web-sdk-eu.aptrinsic.com';
+      break;
+    case 'US2':
+      hostName = 'web-sdk-us2.aptrinsic.com';
+      break;
+  }
+  const sdkUrl= "https://" + hostName + "/api/aptrinsic.js";
+
+  (function(n,t,a,e, co){
+      var i="aptrinsic";
+      n[i]=n[i]||function(){
+        (n[i].q=n[i].q||[]).push(arguments)
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)